services/WorkflowService.ts

```
@@ -0,0 +1,174 @@
+      return true;
+    } catch (error) {
+      console.error("Error deleting workflow:", error);
+      throw error;
+    }
+  }
+
+  static async getWorkflowsByTrigger(triggerId: string): Promise<Workflow[]> {
+    try {
+      const headers = await getHeaders();
+      const response = await axios.get(
+        `${this.BASE_URL}/triggers/${triggerId}`,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)